### ISSUE TITLE
Update conda package to 0.1.7 ?

### ISSUE THREAD

**Issue by daavoo** [pytorch/vision#113]
Calling `conda install torchvision -c soumith` installs version 0.1.6:

```
(pytorch) daviddelaiglesia@daviddelaiglesia:~$ conda install torchvision -c soumith
Fetching package metadata ...........
Solving package specifications: .
Package plan for installation in environment /home/daviddelaiglesia/miniconda3/envs/pytorch:
The following NEW packages will be INSTALLED:
    torchvision: 0.1.6-py35_19 soumith
```

Whereas `pip install torchvision` install the newest version:

```
Collecting torchvision
  Downloading torchvision-0.1.7-py2.py3-none-any.whl
Installing collected packages: torchvision
Successfully installed torchvision-0.1.7
```

Installing pytorch from the homepage binaries (I can confirm this one: `conda install pytorch torchvision cuda80 -c soumith`) also install 0.1.6 torchvision.

I found this because the 0.1.6 version didn't include this fix https://github.com/pytorch/vision/commit/c0a6cfe186301a1733fb926f3a0da89b3283523c

**Comment by daavoo** [comment]
Oh. so the 0.1.7 doesn't include the fix either

**Comment by soumith** [comment]
this fix will be present in `0.1.8` that is being cut tomorrow.

**Comment by daavoo** [comment]
cool!
